packages/mwp-tracking-plugin/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Data collection for MWP apps, including activity and click tracking",
   "engineStrict": true,
   "engines": {
-    "node": ">=8.1.0"
+    "node": "^8.11.4",
+    "yarn": "1.9.4"
```

Comment:
do you think it would be good to be a little looser on the yarn version? e.g. `^1.9.4`, or do you feel like it's pretty fragile?

Reply:
I was considering that. It's probably safe to allow patch updates. I was hoping to keep everyone on the same version as travis uses, but maybe that's not as necessary.